BLSQ/iaso
# IA-2958: Frontend: add permission user management with less powers to the list

Explain what problem this PR is resolving
- Frontend: add permission user management with less powers to the list
Related JIRA tickets : [IA-2958](https://bluesquare.atlassian.net/browse/IA-2958)

## Self proofreading checklist


## Doc
- Tell us where the doc can be found (docs folder, wiki, in the code...)

## Changes
- Add iaso_users_managed permission in modules and PERMISSIONS_PRESENTATION

## How to test
- Edit or create a user or a user role
- Check if the permission **User management - Geo limited** is in the permission's list

## Print screen / video
[Screencast from 2024-05-27 15-07-37.webm](https://github.com/BLSQ/iaso/assets/19631540/5b15a46b-fd54-44d9-a90c-d370649c1adc)



[IA-2958]: https://bluesquare.atlassian.net/browse/IA-2958?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
